.github/workflows/local.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Is this a separate file just so as to keep `ci.yml` identical with the cookiecutter? I would have hoped that it'd be possible/straightforward with `cruft` to have a locally modified version of the file (i.e. some consistent diff against the baseline). This feels like a bit of a step backwards to me.

Reply:
Unfortunately this doesn't seem to be possible right now in cruft. I agree that this is worse, because now we can't cross-reference between the two sets of workflows. Possibly this is something that we can address by using another paradigm in GitHub actions.